### ISSUE TITLE
pilot flagged with malware

### ISSUE THREAD

**Issue by arcologies** [hundredrabbits/Pilot#59]
pilot is being flagged as containing malware, assuming false positive but deleted nonetheless. any thoughts?

**Comment by neauoire** [comment]
Yeah it has some old npm dependencies, I'm not sure if I want to keep maintaining it. I'm nearly done with the Uxn implementation of Pilot, I will likely drop the npm support afterward.

**Comment by leobrowning92** [comment]
Is the way forward for this to switch to the Uxn versions of ORCA/pilot (when ready)? Just started to play around with orca, and pilot provides such a simple, no-fuss way to play with it. Unfortunately my robot overlords aren't happy with the old dependancies. Short of deposing them, are there any other similarly minimal solutions you could point me to?

**Comment by neauoire** [comment]
Hop on #uxn, on irc.esper.net if you need a hand setting things up :)
